common/source/docs/common-auxiliary-functions.rst

```diff
@@ -616,12 +645,84 @@ This three position switch determines if surface tracking via rangefinder is tow
    </td>
    </tr>
    <tr>
-   <tr>
    <td><strong>Standby</strong></td>
    <td>
 
 This puts the autopilot control loops into a soft standby mode so that a parallel, redundant autopilot or companion computer can assume control of the vehicle. The PID loops, position, altitude controllers are modified such that the autopilot can smoothly resume control of the vehicle when standby is subsequently disabled. Switching of outputs or other peripherals must be done by external circuitry.
 
+.. raw:: html
+
+   </td>
+   </tr>
+   <tr>
+   <td><strong>RunCam Control</strong></td>
+   <td>
+
+Allows starting and stopping video recording of compatible RunCam cameras. See :ref:`common-camera-runcam`
+
+.. raw:: html
+
+   </td>
+   </tr>
+   <tr>
+   <td><strong>RunCam OSD Control</strong></td>
+   <td>
+
+Enables control of RunCam cameras OSDs. See :ref:`common-camera-runcam`
+
+.. raw:: html
+
+   </td>
+   </tr>
+   <tr>
+   <td><strong>VISODOM Calibrate</strong></td>
+   <td>
+
+Calibrate VISODOM's camera attitude to vehicles.
```

Comment:
Let's change this one to "align camera's heading to vehicle heading"

Reply:
no problem